background/background.js

```diff
@@ -106,8 +104,8 @@ chrome.runtime.onInstalled.addListener(details => {
             url: chrome.runtime.getURL("/options/options.html")
         })
     } else if (details.reason == "update" && chrome.runtime.getManifest().version != details.previousVersion) {
-        twpConfig.onReady(function () {
-            if (plataformInfo.isMobile.any) return;
+        twpConfig.onReady(async () => {
```

Comment:
Any special reasons to use async here?

Reply:
This function can work better as asynchronous I think